presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestingDatabase.java

```diff
@@ -96,16 +97,16 @@ public JdbcClient getJdbcClient()
 
     public JdbcSplit getSplit(String schemaName, String tableName)
     {
-        JdbcTableHandle jdbcTableHandle = jdbcClient.getTableHandle(new SchemaTableName(schemaName, tableName));
-        JdbcTableLayoutHandle jdbcLayoutHandle = new JdbcTableLayoutHandle(jdbcTableHandle, TupleDomain.all());
+        Optional<JdbcTableHandle> jdbcTableHandle = jdbcClient.getTableHandle(new SchemaTableName(schemaName, tableName));
+        JdbcTableLayoutHandle jdbcLayoutHandle = new JdbcTableLayoutHandle(jdbcTableHandle.orElse(null), TupleDomain.all());
         ConnectorSplitSource splits = jdbcClient.getSplits(jdbcLayoutHandle);
         return (JdbcSplit) getOnlyElement(getFutureValue(splits.getNextBatch(NOT_PARTITIONED, 1000)).getSplits());
     }
 
     public Map<String, JdbcColumnHandle> getColumnHandles(String schemaName, String tableName)
     {
-        JdbcTableHandle tableHandle = jdbcClient.getTableHandle(new SchemaTableName(schemaName, tableName));
-        List<JdbcColumnHandle> columns = jdbcClient.getColumns(session, tableHandle);
+        Optional<JdbcTableHandle> tableHandle = jdbcClient.getTableHandle(new SchemaTableName(schemaName, tableName));
+        List<JdbcColumnHandle> columns = jdbcClient.getColumns(session, tableHandle.orElse(null));
         checkArgument(columns != null, "table not found: %s.%s", schemaName, tableName);
```

Comment:
Not directly related, but this check seems wrong, since `getColumns()` does not return null